packages/backend/src/api.controller.ts

```diff
@@ -1,15 +1,20 @@
-import { Controller, Get } from '@nestjs/common';
+import { Controller, Get, UseGuards } from '@nestjs/common';
 import { ConfigService } from '@nestjs/config';
 import { ApiOkResponse } from '@nestjs/swagger';
+import { DecodedIdToken } from 'firebase-admin/auth';
+import { FirebaseGuard } from './guards/firebase.guard';
 import PingResponse from './schemas/pingResponse';
+import { User } from './util/user.decorator';
 
 @Controller('/api/v1')
+@UseGuards(FirebaseGuard)
 export class APIController {
   constructor(private configService: ConfigService) {}
 
   @Get('ping')
   @ApiOkResponse({ description: 'ping successful', type: PingResponse })
-  getPing(): PingResponse {
+  getPing(@User() user: DecodedIdToken): PingResponse {
```

Comment:
before merging please make user an optional type and return it instead of logging. Might be useful for debugging